CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v14.2.4
```

Comment:
Unless we want to skip versions, this should be 14.2.1. I'm fine either way. In the past, we would just have this version be out of sync with angular/angular.